tests/core/mountpoint/test_base.py

```diff
@@ -396,6 +396,19 @@ async def assert_cannot_write(mountpoint_manager, new_role):
         with pytest.raises(expected_error) as ctx:
             await bar_path.unlink()
 
+        def sync_open():
+            for flag in (
+                os.O_WRONLY,
+                os.O_RDWR,
+                os.O_RDWR | os.O_APPEND,
+                os.O_WRONLY | os.O_NONBLOCK,
```

Comment:
Ooops, `os.O_NONBLOCK` doesn't exist on windows:
https://dev.azure.com/Scille/parsec/_build/results?buildId=7036&view=logs&j=5ba45810-c0e9-5996-abe7-b2cb0c9b6baa&t=0efb1b0d-5afc-53c9-1ac1-4093fcfe20f6&l=73

Here's the list of flags available on both platforms:
```python
os.O_RDONLY
os.O_WRONLY
os.O_RDWR
os.O_APPEND
os.O_CREAT
os.O_EXCL
os.O_TRUNC
```